docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -163,25 +167,29 @@ the configuration are described in the following listings:
 	}
 
 	asciidoctor { <6>
-		inputs.dir snippetsDir <7>
-		dependsOn test <8>
+		dependsOn test <7>
+		configurations 'asciidoctorExt'
+		baseDirFollowsSourceDir() <8>
 	}
 ----
 <1> Apply the Asciidoctor plugin.
-<2> Add a dependency on `spring-restdocs-asciidoctor` in the `asciidoctor` configuration.
-	This will automatically configure the `snippets` attribute for use in your `.adoc`
-	files to point to `build/generated-snippets`. It will also allow you to use the
-	`operation` block macro.
-<3> Add a dependency on `spring-restdocs-mockmvc` in the `testImplementation` configuration. If
+
+<2> Add a dependency on `spring-restdocs-mockmvc` in the `testImplementation` configuration. If
     you want to use `WebTestClient` or REST Assured rather than MockMvc, add a dependency
     on `spring-restdocs-webtestclient` or `spring-restdocs-restassured` respectively
     instead.
+<3> Add a dependency on `spring-restdocs-asciidoctor` in the `asciidoctorExt` configuration.
+	This will automatically configure the `snippets` attribute for use in your `.adoc`
+	files to point to `build/generated-snippets`. It will also allow you to use the
+	`operation` block macro.
 <4> Configure a property to define the output location for generated snippets.
 <5> Configure the `test` task to add the snippets directory as an output.
 <6> Configure the `asciidoctor` task
-<7> Configure the snippets directory as an input.
-<8> Make the task depend on the test task so that the tests are run before the
-	documentation is created.
+<7> Make the task depend on the test task so that the tests are run before the
+    documentation is created.
+<8> The base directory will always be the same as the source directory.
+    If an intermediate working directory is being used,
+    the base directory will automatically point to that.
```

Comment:
AFAIK, this isn't always required so I'd prefer not to document it here.

Reply:
see https://github.com/spring-projects/spring-restdocs/pull/744/files/f027d5263700443b54a3755ca2403d8107a75528#r695151240

Reply:
See https://github.com/spring-projects/spring-restdocs/pull/744#discussion_r702913695

Reply:
done!